install_bootstrap

```diff
@@ -243,6 +245,15 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
         firecloud_import_url='firecloud_import_url'
         ask_question "What is the Launch with FireCloud base url? Unless it's a staging FC enviornment, it should be https://portal.firecloud.org/#import/dockstore" "$FIRECLOUD_IMPORT_URL" "firecloud_import_url" $firecloud_import_url
 
+        sam_authorizer='sam_authorizer'
```

Comment:
I was wondering if you shouldn't call this authorizer_type, since the answer doesn't have to be `sam`. On the other hand, the choice always will be `sam`, so maybe it's OK. Just throwing out there for consideration

Reply:
It's a fair point, I think we may use a hybrid approach if we, say, want to integrate with multiple institutions or inmemory if testing locally. 
I'll create a ticket for follow-up